modern/server/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.8-alpine3.8
```

Comment:
is this server image different from the legacy one?
for server, we don't need any compilers and other stuff, just python and python packages, so I don't see any reason why should we modify it right now

Reply:
Agreed. Done, removed.